build_tools/benchmarks/run_benchmarks.sh

```diff
@@ -12,10 +12,15 @@
 # the script to learn about the required setup.
 #
 # IREE_NORMAL_BENCHMARK_TOOLS_DIR needs to point to a directory contains IREE
-# benchmark tools. See benchmarks/README.md for more information. The first
-# argument is the path of e2e test artifacts directory. The second argument is
-# the path of IREE benchmark run config. The third argument is the path to write
-# benchmark results.
+# benchmark tools. See benchmarks/README.md for more information.
+#
+# Command line arguments:
+# 1. The path of e2e test artifacts directory
+# 2. The path of IREE benchmark run config
+# 3. The target device name
+# 4. The shard index
+# 5. The path to write benchmark results
+# 6. The path to write benchmark traces
```

Comment:
Thanks for the update

Reply:
Hoo boy, maybe we should do some argument parsing

Reply:
Don't shoot the messenger. :innocent: 

I guess we could move the entire script into Python...

Reply:
Lol yeah. Sorry I didn't mean in this PR. I was going to suggest Python but then I looked at the rest of the script and it really is just invoking a couple subprocesses. Of course we *can* do that with Python, but bash feels a bit more natural. And I vaguely recall that we maybe want to be able to run this somewhere where Python isn't installed, but maybe that's old news.